go.mod

```diff
@@ -2,82 +2,96 @@ module github.com/hashicorp/vault-benchmark
 
 go 1.20
 
+replace github.com/hashicorp/vault/sdk => /Users/elliesterner/Developer/hashicorp/vault/sdk
```

Comment:
need to remove this. needed most recent version of sdk to pull in new dockhelper functions